test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+using namespace System.Management.Automation.Internal
```

Comment:
Previously we don't use this in our tests so maybe remove this and use full names in tests.

Reply:
Done.